### PR TITLE
gepifanio/c03-docker06

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Exercise (e.g. C01-E01): c03-docker06

Student Name: gepifanio

> A brief explanation of what has been done.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker06/gepifanio/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.12.0
+ENV NAME=Guilherme
```
Comment (jayanath) [inline classes/03class/exercises/c03-docker06/gepifanio/Dockerfile:2]: Please have a look at this again. The expectation is to pass the name during the docker run, as an argument. The question on README.md is `When the image runs, it should output the following text: Hello `name`, where the value of name is passed during the container execution`

Comment (kikobr82) [reply classes/03class/exercises/c03-docker06/gepifanio/Dockerfile:2]: that's it. Thanks Jay

File: classes/03class/exercises/c03-docker06/gepifanio/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM alpine:3.12.0
+ENTRYPOINT echo Guilherme
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker06/gepifanio/Dockerfile:2]: still not this.
You specify the name during container execution, the command and output should be something as:
```
$ docker run hello-name:v1 Guilherme
Hello Guilherme
```

Comment (gepifanio) [reply classes/03class/exercises/c03-docker06/gepifanio/Dockerfile:2]: Oh ok, hehe got it now.. I will fix it :)

**Pull request merged**